src/api/BundleContent/Bundle.ts

```diff
@@ -24,12 +24,15 @@ import { NodejsappBundlePart } from "./NodejsappBundlePart";
 export class Bundle {
 
   private path = require("path");
+  private fs = require("fs");
```

Comment:
I think the 'TypeScript way' of doing this is 
`import * from as fs from 'fs'`

Reply:
Typescript hasn't complained. Maybe there's more than one way to do it?